CollAction/Migrations/20180329050159_ProjectRichText.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
There're multiple migrations for this pull request. Could you compress them into one migration? Adding and then removing columns might result in a suboptimal disk-structure for the tables with some database systems.